doc/whats_new/v1.0.rst

```diff
@@ -45,6 +45,37 @@ Changelog
     :pr:`123456` by :user:`Joe Bloggs <joeongithub>`.
     where 123456 is the *pull request* number, not the issue number.
 
+- |Feature| The option for using the squared error via ``loss`` and
```

Comment:
Shouldn't we tag this entry as `|API|`? Usually deprecation warnings use the `|API|` header.